samples/wifi/sta/README.rst

```diff
@@ -23,6 +23,14 @@ This sample can perform Wi-Fi operations such as connect and disconnect in the 2
 
 Using this sample, the development kit can connect to the specified access point in :abbr:`STA (Station)` mode.
 
+Quad Serial Peripheral Interface (QSPI) encryption
+**************************************************
+
+This sample enables QSPI encryption by default using the factory default key (all F's), but you can change the key by setting the :kconfig:option:`CONFIG_NRF700X_QSPI_ENCRYPTION_KEY` Kconfig option,
+this demonstrates only the API usage.
```

Comment:
```suggestion
This Kconfig option demonstrates only the API usage.
```

Reply:
Please ensure that this suggested edit does not change the intended meaning of this sentence.
I am not sure if it is the Kconfig option mentioned that demonstrates API usage, or the key.

Reply:
Its not the Kconfig, it should be `this sample demonstrated only the API usage`.